features/receive_vote.feature

```diff
@@ -1,4 +1,4 @@
 Feature: Receive Vote
   As a developer
   So that I can give my input on story complexity/difficulty
-  I would like to receive information about an async vote including URL, title and voting/discussion options
+  I would like to receive information about an async vote
```

Comment:
I'm not sure we want to remove this - I'd say the key thing was to change the name of the feature to "Lookup current vote"